### PR TITLE
added profile

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]
good code structure, can be merged

File: fashify/src/pages/Profile/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          <NavLink to={'/profile/details'} className='profile-nav-item'>
+            Profile Information
+          </NavLink>
+          {/*address management and order management will be added later on*/}
+          {/* <NavLink to={'/profile/addresses'} className='profile-nav-item'>
+            Addresses
+          </NavLink>
+          <NavLink to={'/profile/orders'} className='profile-nav-item'>
+            Order History
+          </NavLink> */}
```
Comment (romabulani) [inline fashify/src/pages/Profile/Profile.jsx:18]: Remove the comments if not required

File: fashify/src/pages/Auth/Signup/Signup.jsx [hunk 2]
```diff
@@ -128,12 +116,18 @@ const SignUp = () => {
...
+            <div className='auth-footer'>
+              <p>
+                <span>Already have an account?</span>
+                <span
+                  className='auth-signup'
+                  onClick={() => navigate('/login')}
+                >
+                  {' '}
+                  sign in
+                </span>
```
Comment (romabulani) [inline fashify/src/pages/Auth/Signup/Signup.jsx]: Here, just a suggestion, you can use Link, you should avoid use of navigate as much as possible

**Review by romabulani — APPROVED** [review]
- Modular code
- Have given some suggestions if you have time
- good to merge

**Review by anuj-kumary — APPROVED** [review]
The code looks clean and good to merge

File: fashify/src/pages/Profile/components/Details/Details.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import './Details.css';
+
+export const Details = () => {
+  const { currUser: user, logoutHandler } = useAuth();
+  const navigate = useNavigate();
+  const ProfileLogoutHandler = () => {
+    logoutHandler();
+    navigate('/logout');
+  };
+  console.log(user);
```
Comment (VishalPatil18) [inline fashify/src/pages/Profile/components/Details/Details.jsx]: You could remove these unwanted `console.log()`'s

File: fashify/src/pages/Auth/Signup/Signup.jsx [hunk 4]
```diff
@@ -128,12 +116,18 @@ const SignUp = () => {
...
-              Already have an account
-            </button>
+            <div className='auth-footer'>
+              <p>
+                <span>Already have an account?</span>
+                <span
+                  className='auth-signup'
+                  onClick={() => navigate('/login')}
+                >
+                  {' '}
```
Comment (VishalPatil18) [inline fashify/src/pages/Auth/Signup/Signup.jsx:123]: You could remove the unwanted `{' '}`

File: fashify/src/pages/Auth/Login/Login.jsx [hunk 5]
```diff
@@ -91,12 +72,18 @@ const Login = () => {
...
-              Create New Account
-            </button>
+            <div className='auth-footer'>
+              <p>
+                <span>Don't have an account?</span>
+                <span
+                  className='auth-signup'
+                  onClick={() => navigate('/signup')}
+                >
+                  {' '}
```
Comment (VishalPatil18) [inline fashify/src/pages/Auth/Login/Login.jsx:79]: You could remove the unwanted `{' '}`

File: fashify/src/pages/Auth/Login/Login.jsx [hunk 6]
```diff
@@ -91,12 +72,18 @@ const Login = () => {
...
-            <button
-              onClick={() => navigate('/signup')}
-              className='btn btn-link-primary outlined-secondary text-align-center brd-rd-semi-sq'
-            >
-              Create New Account
-            </button>
+            <div className='auth-footer'>
+              <p>
+                <span>Don't have an account?</span>
+                <span
```
Comment (VishalPatil18) [inline fashify/src/pages/Auth/Login/Login.jsx]: You can use the `<Link>` here rather than a span to avoid using navigate explicitly.

**Review by VishalPatil18 — APPROVED** [review]
A few suggestions were added. The rest looks quite good and works well. Great work @Rohanmond 👍

**Review by GauravDe-git — APPROVED** [review]
- Clean and well-formatted code
- Props to Vishal for the great suggestions, apart from them, I found as such no other issues.

**Review by vinc3nati — APPROVED** [review]

**Pull request merged**